analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SpecifyIFormatProviderOrCultureInfo.CSharp11.cs

```diff
@@ -12,4 +12,14 @@ void TestCases()
             int.TryParse("123", out result); // Noncompliant
         }
     }
+
+    // https://github.com/SonarSource/sonar-dotnet/issues/8233
+    class Repro_8233
+    {
+        public void IgnoreCulture()
+        {
+            // Need more example of type implementating IParseable that ignore culture
+            _ = Guid.Parse(""); // Noncompliant - FP
```

Comment:
We may add some more candidates that are controversial in the file with some comment what we should do about it.
```suggestion
            _ = Guid.Parse(""); // Noncompliant - FP
            _ = int.Parse("-1"); // Noncompliant - FP Controversial: There are edge cases like culture "fa-AF" where the 0x2212 (MINUS SIGN) is used instead of the usual 0x002D (HYPHEN-MINUS)